docs/code-review.md

```diff
@@ -0,0 +1,61 @@
+# Code Review Checklist
+
+Currently, all changes to the product must be reviewed by a [core committer](/contribute/getting-started/core-committers/#core-committers).
+
+<!-- * Documentation changes must be reviewed by a [product manager](/contribute/getting-started/core-committers/#product-managers).
```

Comment:
Is this intentionally commented out?  While these comments work on GH, they don't work for all markdown parses.  IMO we should uncomment or delete.